performance_test/src/experiment_configuration/communication_mean.hpp

```diff
@@ -40,6 +43,10 @@ inline std::ostream & operator<<(std::ostream & stream, const CommunicationMean
 {
   if (cm == CommunicationMean::ROS2) {
     return stream << "ROS2";
+#ifdef PERFORMANCE_TEST_POLLING_SUBSCRIPTION_ENABLED
+  } else if (cm == CommunicationMean::ROS2PollingSubscription) {
+    return stream << "ROS2PollingSub";
```

Comment:
so here also ROS2PollingSubscription for consistency

Reply:
Okay changed this and also changed here as the name of the command line option : https://github.com/ApexAI/performance_test/blob/3290-ros2-waitset/performance_test/src/experiment_configuration/experiment_configuration.cpp#L145 and updated the description here :  https://github.com/ApexAI/performance_test/pull/88#issue-319478231 to reflect the same.